src/hooks/useTasks/useTasks.tsx

```diff
@@ -21,46 +23,51 @@ type UseTasks = (props?: UseTasksProps) => {
   setCurrentDate: (date: Date) => void
   dateRange: { startDate: Date, endDate: Date }
   setDateRange: (props: { startDate: Date, endDate: Date }) => void
-  refetch: () => void
+  refetch: () => Promise<{ data: Task[] } | undefined>
   submitResponseToTask: (taskId: string, userResponse: string) => void
   uploadDocumentsForTask: (taskId: string, files: File[], description?: string) => Promise<void>
   deleteUploadsForTask: (taskId: string) => void
   updateDocUploadTaskDescription: (taskId: string, userResponse: string) => void
+  unresolvedTasks?: number
 }
 
 type UseTasksProps = {
-  startDate?: Date,
+  startDate?: Date
   endDate?: Date
 }
 
 const DEBUG_MODE = false
 
 export const useTasks: UseTasks = ({
   startDate: initialStartDate = startOfYear(new Date()),
-  endDate: initialEndDate = endOfYear(new Date())
+  endDate: initialEndDate = endOfYear(new Date()),
 }: UseTasksProps = {}) => {
   const [loadedStatus, setLoadedStatus] = useState<LoadedStatus>('initial')
 
-  const { businessId, read, syncTimestamps, hasBeenTouched } = useLayerContext()
+  const { business, businessId, read, syncTimestamps, hasBeenTouched } = useLayerContext()
+  const activationDate = getActivationDate(business)
 
   const { apiUrl } = useEnvironment()
   const { data: auth } = useAuth()
 
   const [dateRange, setDateRange] = useState({
     startDate: initialStartDate,
-    endDate: initialEndDate
+    endDate: initialEndDate,
   })
   const [currentDate, setCurrentDate] = useState(new Date())
 
-  const queryKey = businessId && auth?.access_token && `tasks-${businessId}-${dateRange.startDate}-${dateRange.endDate}`
+  const queryKey = businessId && activationDate && auth?.access_token && `tasks-${businessId}-${activationDate.toISOString()}`
 
   const { data, isLoading, isValidating, error, mutate } = useSWR(
     queryKey,
     Layer.getTasks(apiUrl, auth?.access_token, {
       params: {
         businessId,
-        startDate: formatISO(dateRange.startDate.valueOf()),
-        endDate: formatISO(dateRange.endDate.valueOf()),
+        // startDate: formatISO(dateRange.startDate.valueOf()),
+        // endDate: formatISO(dateRange.endDate.valueOf()),
+        /** Get the whole history of tasks */
+        startDate: activationDate ? formatISO(activationDate.valueOf()) : undefined,
+        endDate: formatISO((new Date()).valueOf()),
```

Comment:
Now we read the whole history so we know if there is any unresolved task in other than current year